src/auxiliary/p4est.jl

```diff
@@ -24,6 +24,10 @@ function init_p4est()
 end
 
 
+# Check if p4est library supports MPI
+@inline p4est_has_mpi() = isdefined(P4est, :P4EST_ENABLE_MPI)
```

Comment:
Ah, thanks for reminding me of this. I absolutely think it would be great to have this function in P4est.jl instead. Should I also add this in https://github.com/trixi-framework/P4est.jl/pull/45 ?

Reply:
Yeah, sounds good.

Reply:
Replaced this function with P4est.jl's new built-in version in 096b0a3.

Reply:
Great!